plugins/nodes/src/js/types/Status.ts

```diff
@@ -1,20 +1,45 @@
 import { i18nMark } from "@lingui/react";
 import StatusIcon from "#SRC/js/constants/StatusIcon";
+import Node from "#SRC/js/structs/Node";
+
+const build = (priority: number, icon: StatusIcon, name: string): Status => ({
+  priority,
+  icon,
+  displayName: i18nMark(name)
+});
+
+const active = build(2, StatusIcon.SUCCESS, "Active");
+const deactivated = build(3, StatusIcon.STOPPED, "Deactivated");
+const drained = build(5, StatusIcon.STOPPED, "Drained");
+const draining = build(4, StatusIcon.LOADING, "Draining");
+const unknown = build(1, StatusIcon.ERROR, "Unknown");
```

Comment:
Shouldn't 1 be the highest priority? Also, a small thing, but I think it would help with readability if these were sorted.